.github/workflows/autolabel.yml

```
@@ -0,0 +1,15 @@
+on:
+    issues:
+        type: [opened]
+
+jobs:
+    autolabel:
+      runs-on: ubuntu-latest
+      name: auto label
+      steps:
+      - name: issuelabeler
+        uses: docker://maxandersen/jbang-issuelabeler
+        env:
+          GITHUB_TOKEN: ${{ secrets.ISSUE_GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
```
(this is the one we have)